advanced-usage/index.md

```diff
@@ -141,12 +186,14 @@ This is primarily useful for underwater vehicles.
 Configuration determines whether the exact depth value is shown, and the colour of the lines:
 {{ easy_image(src="depth-hud-config", width=500, center=true) }}
 
-##### Indicators
+##### IFrame
+
+The iframe widget provides an inline frame that can display another HTML page within the Cockpit interface.
+This is particularly useful for showing the interfaces for BlueOS Extensions:
+{{ easy_image(src="iframe-widget", width=400, center=true) }}
 
-The indicators widget shows a collection of telemetry values, including the vehicle's attitude, GPS location,
-and optionally its power supply and usage and some debugging data: 
-{{ easy_image(src="indicators-widget", width=200, center=true) }}
-{{ easy_image(src="indicators-config", width=300, center=true) }}
+Configuration determines the URL to fetch the page from, as well as the overall transparency of the iframe:
+{{ easy_image(src="iframe-config", width=400, center=true) }}
```

Comment:
"a common usage is to place an IFrame widget on the view and point it to the sonar view page of a sonar extension"